MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -137,6 +137,13 @@
 #include <setjmp.h>
 #include <unistd.h>
 
+jmp_buf jbAnagram;
+
+void *memset(void *s : byte_count(n), int c, size_t n) : bounds(s, (char *) s + n);
```

Comment:
Can you explain why stdlib_checked.h has __FORTIFY_SOURCE flags?
By default, __FORTIFY_SOURCE flags is unset
In lnt runtest script, how can we provide additional cflags -D_FORTIFY_SOURCE?